Client/src/Pages/Monitors/CreateMonitor/index.jsx

```diff
@@ -124,46 +124,58 @@ const CreateMonitor = () => {
 
   const handleCreateMonitor = async (event) => {
     event.preventDefault();
-    //obj to submit
-    let form = {
-      url:
-        //preprending protocol for url
-        monitor.type === "http"
-          ? `http${https ? "s" : ""}://` + monitor.url
-          : monitor.url,
-      name: monitor.name === "" ? monitor.url : monitor.name,
-      type: monitor.type,
-      interval: monitor.interval * MS_PER_MINUTE,
-    };
 
-    const { error } = monitorValidation.validate(form, {
-      abortEarly: false,
-    });
+    const formattedUrl = monitor.type === "http"
+    ? `http${https ? "s" : ""}://` + monitor.url
+    : monitor.url;
 
-    if (error) {
-      const newErrors = {};
-      error.details.forEach((err) => {
-        newErrors[err.path[0]] = err.message;
-      });
-      setErrors(newErrors);
-      createToast({ body: "Error validation data." });
-    } else {
-      form = {
-        ...form,
-        description: form.name,
-        teamId: user.teamId,
-        userId: user._id,
-        notifications: monitor.notifications,
+    try {
+      // Check if the URL resolves by sending a request
+      const response = await fetch(formattedUrl, { method: 'GET', mode: 'no-cors' });
```

Comment:
Checking if the URL resolves should be done on the back end, sending a `no-cors` request from the front end won't tell us if the server we want to reach is reachable or not.

The response is opaque, so we have no idea what is in the response, just that we got something back.  If there was a proxy inbetween us and the server for example we could still get a 200 response, but the server might not be reachable.

We should implement a more comprehensive solution which would look like this:

Front End sends GET request to Back End -> BE forwards request to Server URL -> BE returns response to FE